sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -215,7 +215,13 @@ template <> inline void print<>(PiResult val) {
 }
 
 // cout does not resolve a nullptr.
-template <> inline void print<>(std::nullptr_t val) { print<void *>(val); }
+template <> inline void print<>(std::nullptr_t val) {
+  std::cout << "<nullptr> : " << static_cast<void *>(val) << std::endl;
+}
+
+template <> inline void print(char *val) {
```

Comment:
Any particular reason you omitted `<>` after `print` here?

Reply:
Nope.  Cut and paste error.  I will add it.

Reply:
done.